dkms.conf

```diff
@@ -0,0 +1,38 @@
+PACKAGE_NAME="ivsc-driver"
+PACKAGE_VERSION="1.0.0"
+
+MAKE="make all"
+CLEAN="make clean"
```

Comment:
Should always respect `$kernelver` and `$kernel_source_dir`:
```
MAKE="make KERNELRELEASE=$kernelver KERNEL_SRC=$kernel_source_dir"
CLEAN="make KERNELRELEASE=$kernelver KERNEL_SRC=$kernel_source_dir clean"
```

Reply:
where does kernelver and kernel_source_dir come from? can you please revisit? @vicamo